collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java

```diff
@@ -683,7 +684,7 @@ public void onClick(DialogInterface dialog, int i) {
     }
 
     private void createProgressDialog() {
-        progressDialog = new ProgressDialog(this);
+        progressDialog = new AlertDialog.Builder(this).create();
```

Comment:
Could this not use a `ProgressDialogFragment` instead?

Reply:
Oh I didn't use `ProgressDialogFrgament` because I thought that the lifecycle changes are handled here by the `ViewModel`, so, I just used an `AlertDialog` instead. But, as I have read  in most of the articles like [here](https://guides.codepath.com/android/using-dialogfragment), that using a dialog instead of fragment, and handling lifecycle changes by the use of some variable, is a bad practice, so I'll just switch to ProgressDialogFragment here.